tools/cli/admin.go

```diff
@@ -683,6 +683,36 @@ func newAdminTaskListCommands() []cli.Command {
 				AdminDescribeTaskList(c)
 			},
 		},
+		{
+			Name:  "list_tasks",
+			Usage: "List tasks of a tasklist",
+			Flags: append(getDBFlags(),
+				cli.StringFlag{
+					Name:  FlagNamespaceID,
+					Usage: "Namespace Id",
+				},
```

Comment:
Lets add field for TaskID to be able to select a singular based off Friday disussion

Reply:
created a tracking ticket to support predicates on all `list` commands https://app.clubhouse.io/temporal-technologies-inc/story/845/support-predicates-for-admin-list-commands